vici/requests.go

```diff
@@ -124,6 +127,10 @@ func (s *Session) streamEventRegisterUnregister(event string, register bool) err
 // dedicated transport. It should only be used within a function with the session
 // lock.
 func (s *Session) cmdTransportCommunicate(pkt *packet) (*packet, error) {
+	if s.ctr == nil {
```

Comment:
Same thing here (but for `sendRequest`).

Reply:
done